README.md

```diff
@@ -367,107 +367,109 @@ rate](https://gitlab.com/ccao-data-science---modeling/models/ccao_res_avm/-/blob
-| Sale Day                                                                | Time           | numeric     |                                                                      | Sale day calculated as the number of days since January 1st, 1997                                                                                     |
-| Sale Quarter of Year                                                    | Time           | character   |                                                                      | Character encoding of quarter of year (Q1 - Q4)                                                                                                       |
-| Sale Month of Year                                                      | Time           | character   |                                                                      | Character encoding of month of year (Jan - Dec)                                                                                                       |
-| Sale Day of Year                                                        | Time           | numeric     |                                                                      | Numeric encoding of day of year (1 - 365)                                                                                                             |
-| Sale Day of Month                                                       | Time           | numeric     |                                                                      | Numeric encoding of day of month (1 - 31)                                                                                                             |
-| Sale Day of Week                                                        | Time           | numeric     |                                                                      | Numeric encoding of day of week (1 - 7)                                                                                                               |
-| Sale After COVID-19                                                     | Time           | logical     |                                                                      | Indicator for whether sale occurred after COVID-19 was widely publicized (around March 15, 2020)                                                      |
+powerfully predictive, and well-behaved.
+
+For a machine-readable version of this data dictionary, see
+[`docs/data-dict.csv`](./docs/data-dict.csv).
+
+| Feature Name                                                            | Variable Name                                         | Description                                                                                                                                           | Category       | Possible Values                                                      |
```

Comment:
Big diff here, mainly because I've rearranged the column order. I double-checked to make sure the parameter count matches `params.yaml`.